package.json

```diff
@@ -1,81 +1,39 @@
 {
-  "name": "reatom",
-  "private": true,
-  "description": "Expressive and deterministic state manager",
+  "name": "reatom-workspace",
   "scripts": {
     "release": "npx tsx ./tools/publish.ts",
-    "postinstall": "turbo run build --filter=core --filter=framework --filter=testing",
-    "build": "turbo run build",
-    "reset": "./reset.sh",
-    "test": "turbo run build --filter=testing && vitest",
-    "test:eslint-plugin": "turbo run test --filter=@reatom/eslint-plugin",
+    "postinstall": "turbo run build --output-logs=errors-only --filter=./packages/{framework,testing}",
+    "build": "turbo run build --output-logs=errors-only --filter=./packages/*",
+    "reset": "sh ./tools/reset.sh",
+    "test": "turbo run build --output-logs=errors-only --filter=./packages/testing",
     "prettify": "prettier --write */**/**/*.{js,ts,md}",
     "prettify:check": "prettier --check */**/**/*.{js,ts,md}",
     "prettify:watch": "onchange */**/**/*.{js,ts,md} -- prettier --write {{changed}}",
     "package-generator": "tsx tools/new-package.ts",
     "update": "npx npm-check-updates -u",
-    "prepare": "git config blame.ignoreRevsFile .git-blame-ignore-revs && husky install",
+    "prepare": "husky",
     "actualize-issue-templates": "npx tsx ./tools/actualize-issue-templates.ts"
   },
   "workspaces": [
-    "packages/*"
+    "./packages/*"
   ],
-  "author": "artalar",
-  "license": "MIT",
-  "readme": "README.md",
-  "repository": {
-    "type": "git",
-    "url": "git+ssh://git@github.com/artalar/reatom.git"
-  },
+  "packageManager": "npm@8.0.0",
   "engines": {
     "npm": ">=8.0.0"
   },
-  "bugs": {
-    "url": "https://github.com/artalar/reatom/issues"
-  },
-  "homepage": "https://github.com/artalar/reatom/tree/v3",
```

Comment:
This can be usefull for some tooling which parse the root package.json and don't support monorepos